sopel/irc/isupport.py

```diff
@@ -131,10 +131,37 @@ def _parse_prefix(value):
 }
 
 
+def _hex_unescape(param):
+    """Unescape hexadecimal escape sequences (i.e. ``'\\xHH'``)
```

Comment:
Change this to e.g. not i.e.

https://www.grammarly.com/blog/know-your-latin-i-e-vs-e-g/

Reply:
I think "i.e." is appropriate, as it only escapes things in the form `\xHH`

Reply:
then the comment before should be more specific. "escape sequences" is too general

Reply:
Sure, but "hexadecimal escape sequences" as written is sufficiently specific imo :p

Reply:
The grammar here matches my intent, but the name/doc can be a little more explicit, change made in fa45f08